src/group_theory/order_of_element.lean

```diff
@@ -146,6 +146,9 @@ by_contradiction
     from nat.mod_lt _ (order_of_pos _))
       ⟨nat.pos_of_ne_zero (mt nat.dvd_of_mod_eq_zero h₁), by rwa ← pow_eq_mod_order_of⟩)
 
+lemma order_of_dvd_iff_pow_eq_one {n : ℕ} : a ^ n = 1 ↔ order_of a ∣ n :=
```

Comment:
The name of the lemma should match the statement, in the same order. Either exchange the two sides of the equivalence, or the two sides of the name.

Reply:
aargh sorry! Rookie error. I even asked Chris which way round I should write it :-)